package.json

```diff
@@ -97,6 +97,7 @@
     "@storybook/addons": "^6.0.21",
     "@storybook/react": "^6.0.21",
     "@storybook/theming": "^6.0.21",
+    "@testing-library/dom": "^7.21.4",
```

Comment:
This was missing, unrelated to the PR (i can move it to another PR if necessary)